src/modules/priorities/application/use-cases/swap-server-priorities.use-case.ts

```
@@ -0,0 +1,99 @@
+import { Injectable, NotFoundException, ForbiddenException } from '@nestjs/common';
```

ESLint

[error] Replace `·Injectable,·NotFoundException,·ForbiddenException·` with `⏎··Injectable,⏎··NotFoundException,⏎··ForbiddenException,⏎`

(prettier/prettier)